valohai_yaml/schema/execution-resources.yaml

```diff
@@ -0,0 +1,29 @@
+type: object
+description: Resource requirements and limits for the endpoint.
```

Comment:
the description probably either needs to be modified case by case or made more generic for Kubernetes workers?

Reply:
In that case, it might make more sence to just generalize the resource properties instead of the whole resources node?

Reply:
I'm a bit pessimistic  that these descriptions are actually read that much TBH so I would just go with a generalized description 🤔 But you can go either way, both approaches are easy to change if need-be 😆

Reply:
both work!

what I would advise against is copy-pasting the definitions, as we probably want parity in the naming of the properties for both use cases to be the same

Reply:
Refactored this to have separate references for all properties.
Still requires naming the properties in each schema though but I don’t think there’s a way to avoid this that would be easier and more clear than just copying the same property names to each schema that uses these.

This allows, however, setting different resource descriptions and using a different combination of properties in different schemas.